runtime/src/lib.rs

```diff
@@ -1673,20 +1663,15 @@ impl_runtime_apis! {
 			add_benchmark!(params, batches, pallet_indices, Indices);
 			add_benchmark!(params, batches, pallet_membership, TechnicalMembership);
 			add_benchmark!(params, batches, pallet_multisig, Multisig);
-	 //     add_benchmark!(params, batches, pallet_offences, OffencesBench::<Runtime>);
 			add_benchmark!(params, batches, pallet_proxy, Proxy);
 			add_benchmark!(params, batches, pallet_scheduler, Scheduler);
-	 //       add_benchmark!(params, batches, pallet_session, SessionBench::<Runtime>);
 			add_benchmark!(params, batches, pallet_staking, Staking);
 			add_benchmark!(params, batches, pallet_democracy, Democracy);
 			add_benchmark!(params, batches, frame_system, SystemBench::<Runtime>);
 			add_benchmark!(params, batches, pallet_timestamp, Timestamp);
 			add_benchmark!(params, batches, pallet_treasury, Treasury);
 			add_benchmark!(params, batches, pallet_utility, Utility);
-			//TODO: [#463] Currently broken, will be fixed in different issue
-//			add_benchmark!(params, batches, pdex_migration, PDEXMigration);
-			// add_benchmark!(params, batches, asset_handler, AssetHandler);
-			if batches.is_empty() { return Err("Benchmark not found for this pallet.".into()) }
+*/			if batches.is_empty() { return Err("Benchmark not found for this pallet.".into()) }
```

Comment:
Same

Reply:
historic reasons. those are included from substrate's frame are benched by parity. @Gauthamastro can I clean up commented ones? we will not re-bench them.

Reply:
same